pkg/s3utils/s3utils.go

```diff
@@ -28,6 +29,26 @@ type Object struct {
 	LastModified time.Time
 }
 
+type SSE_TYPE string
+
+const (
+	SSE_S3  SSE_TYPE = "SSE-S3"
+	SSE_KMS SSE_TYPE = "SSE-KMS"
+	SSE_C   SSE_TYPE = "SSE-C"
```

Comment:
Use go conventions and camel case everywhere

Reply:
done